openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -37,13 +34,49 @@ def create(self, subset_name, instance_data, pre_create_data):
         }
 
         if self.selected_nodes:
-            parms["sop_path"] = self.selected_nodes[0].path()
+            selected_node = self.selected_nodes[0]
+
+            # Although Houdini allows ObjNode path on `sop_path` for the
+            # the ROP node we prefer it set to the SopNode path explicitly
+
+            # Allow sop level paths (e.g. /obj/geo1/box1)
+            # but do not allow other sop level paths when
+            #   the parent type is not 'geo' like
+            #   Cameras, Dopnet nodes(sop solver)
+            if isinstance(selected_node, hou.SopNode) and \
+                selected_node.parent().type().name() in ['geo', 'subnet']:
+                parms["sop_path"] = selected_node.path()
+                self.log.debug(
+                   "Valid SopNode selection, 'SOP Path' in ROP will be set to '%s'."
+                   % selected_node.path()
+                )
+
+            # Allow object level paths to Geometry nodes (e.g. /obj/geo1)
+            # but do not allow other object level nodes types like cameras, etc.
+            elif isinstance(selected_node, hou.ObjNode) and \
+                    selected_node.type().name() in ['geo']:
```

Comment:
can be also this.
```suggestion
                    selected_node.type().name() == "geo":
```

Reply:
Sure, however I did it to be more consistent and to make it easier to add more allowed types. 
e.g. 
```python
selected_node.parent().type().name() in ["geo", "subnet"]:
```